src/manage_sql_report_formats.c

```diff
@@ -3997,11 +3997,18 @@ apply_report_format (gchar *report_format_id,
 
   rf_dependencies_string
```

Comment:
A bit late but this could have an `if (report_config)` around it, because the report_config is optional. It would save the SQL query when the report_config is 0 (and maybe prevent some weird situations if someone somehow has a report_config with id 0).